query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/filters/field_reference/json_filter.rs

```diff
@@ -1,6 +1,6 @@
 use query_engine_tests::*;
 
-#[test_suite(capabilities(JsonFiltering), exclude(MySql(5.6)))]
+#[test_suite(capabilities(JsonFiltering), exclude(Mysql56))]
```

Comment:
Without this change, the test suite [would fail](https://buildkite.com/prisma/test-prisma-engines/builds/22705#018d1d05-6ae2-4654-811a-620c333abff5) on `mysql:5.6`.